.github/workflows/test.yaml

```diff
@@ -3,11 +3,15 @@ on: [pull_request]
 jobs:
   run_tests:
     runs-on: ubuntu-22.04
-    container: golang:1.20
     steps:
       -
         name: Checkout
         uses: actions/checkout@v2
+
+      - uses: actions/setup-go@v3
```

Comment:
I am generally not a fan of these setup steps as it makes the workflow environment even harder to reproduce locally, however 👍🏼